blocks/recent-posts/recent-posts.css

```diff
@@ -0,0 +1,98 @@
+.blog main .block.recent-posts[data-block-status="loaded"] {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+}
+
+.blog main .block.recent-posts[data-block-status="loaded"] .blog-card > .card-content > .blog-link h5 {
+    font-family: 'Playfair Display', serif;
+    font-size: 18px;
+    color: var(--text-color);
+  }
+
+  .blog main .block.recent-posts[data-block-status="loaded"] .blog-card > .card-content:hover > .blog-link h5 {
```

Comment:
After changing "h5" to "h3" in your recent-posts.js, then you will need to change all your "h5" references to "h3" in the CSS files, such as this one.

Reply:
fixed